qt_ui/windows/basemenu/QBaseMenu2.py

```diff
@@ -240,10 +241,12 @@ def closeEvent(self, close_event: QCloseEvent):
         GameUpdateSignal.get_instance().updateGame(self.game_model.game)
 
     def get_base_image(self):
-        if self.cp.cptype == ControlPointType.AIRCRAFT_CARRIER_GROUP:
-            return "./resources/ui/carrier.png"
-        elif self.cp.cptype == ControlPointType.LHA_GROUP:
-            return "./resources/ui/lha.png"
+        if (
+            self.cp.cptype == ControlPointType.AIRCRAFT_CARRIER_GROUP
+            or self.cp.cptype == ControlPointType.LHA_GROUP
+        ):
+            carrier_type = self.cp.get_carrier_group_type()
```

Comment:
If super carrier option is not enabled this will always show the base carrier (stennis) and not the one representing the Carrier chosen by the Campaign (even if it is just a name) cp.name i think.

Reply:
I have now changed this so it works like in your latter option.